src/main/java/io/github/queerbric/queerbrics/mixin/ItemMixin.java

```diff
@@ -29,12 +29,13 @@
 	public void use(World world, PlayerEntity user, Hand hand, CallbackInfoReturnable<TypedActionResult<ItemStack>> info) {
 		ItemStack itemStack = user.getStackInHand(hand);
 		if (itemStack.getItem() == Items.BRICK || itemStack.getItem() == Items.NETHER_BRICK) { // TODO tag for supporting other items?
-			world.playSound((PlayerEntity)null, user.getX(), user.getY(), user.getZ(), SoundEvents.ENTITY_SNOWBALL_THROW, SoundCategory.NEUTRAL, 0.5F, 0.4F / (RANDOM.nextFloat() * 0.4F + 0.8F));
+			world.playSound(null, user.getX(), user.getY(), user.getZ(), QueerbricsSoundEvents.brickThrow, SoundCategory.NEUTRAL, 1.2F, 0.7f);
 			if (!world.isClient) {
 				ThrowableBrickEntity throwableBrickEntity = new ThrowableBrickEntity(world, user);
 				throwableBrickEntity.setItem(itemStack);
 				throwableBrickEntity.setProperties(user, user.pitch, user.yaw, 0.0F, 1.5F, 1.0F);
 				world.spawnEntity(throwableBrickEntity);
+				itemStack.decrement(1);
```

Comment:
This should not be decremented if the player has creative-mode abilites

Reply:
This is decrement's vanilla behavior, it will not decrement in creative